src/oas2/transformers/__tests__/params.test.ts

```diff
@@ -193,7 +195,6 @@ describe('params.translator', () => {
           {},
           {
             in: 'body',
-            name: 'name',
```

Comment:
I removed this to test the case if name wasn't supplied in the original document.